prebuilt/external/snowball/libstemmer/modules.h

```diff
@@ -0,0 +1,265 @@
+/* /work/ArangoDB/build/3rdParty/iresearch/external/snowball/libstemmer/modules.h: List of stemming modules.
```

Comment:
I don't think we do need to use pre-built stuff from snowball as it doesn't bring any additional dependencies